### PR TITLE
[Task-7][Frontend] As a user, I can login

### LINKED ISSUE #7: [Task-7][Frontend] As a user, I can login
## Why

Refined Frontend for better user experience
 
## Acceptance Criteria
- [ ] Create login frontend form with bootstrap

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-61389950

## What happened 👀

Design login page.
 
## Insight 📝

- Design login page to make it look better with bootstrap.
- Make toaster absolute
 
## Proof Of Work 📹

image

### CODE REVIEW

File: lib/elixir_search_extractor_web/templates/shared/_app-toast.html.eex [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+			<a class="app-toast__close" data-bs-dismiss="alert" aria-label="close">&times;</a>
+		</div>
+	<% end %>
+
+	<%= unless is_nil(get_flash(@conn, :error)) do %>
+	<div class="alert alert-danger fade show app-toast__alert">
+		<p class="app-toast__message"><%= get_flash(@conn, :error) %></p>
+		<a class="app-toast__close" data-bs-dismiss="alert" aria-label="close">&times;</a>
+	</div>
+	<% end %>
```
Comment (rosle) [inline lib/elixir_search_extractor_web/templates/shared/_app-toast.html.eex]: Let's fix the indentation here 😆

**Review by rosle — APPROVED** [review]

File: lib/elixir_search_extractor_web/templates/shared/_app-toast.html.eex (hunk 1 above)
Comment (rafayet-monon) [reply lib/elixir_search_extractor_web/templates/shared/_app-toast.html.eex]: @rosle  Fixed issues in [337cc03](https://github.com/rafayet-monon/elixir_search_extractor/pull/29/commits/337cc03a2eb69e1a0c06ede52357ba4c3fdff9fa)

File: assets/css/components/_app-toast.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  &__alert {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__message {
+    margin: 0;
+  }
+
+  &__close {
```
Comment (hanam1ni) [inline assets/css/components/_app-toast.scss]: The current name is a bit abstract, let's rename it to be more component maybe `&__close-btn` or  `&__dismiss-btn`

Comment (rafayet-monon) [reply assets/css/components/_app-toast.scss]: Added in [9bcc594](https://github.com/rafayet-monon/elixir_search_extractor/pull/29/commits/9bcc594b71e75142d8612d62a008febd413e6c66)

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**